samza-core/src/test/java/org/apache/samza/task/TestAsyncRunLoop.java

```diff
@@ -368,8 +367,10 @@ public void testCommitSingleTask() throws Exception {
     task0ProcessedMessagesLatch.await();
     task1ProcessedMessagesLatch.await();
 
-    verify(offsetManager).checkpoint(taskName0);
-    verify(offsetManager, never()).checkpoint(taskName1);
+    verify(offsetManager).createCheckpoint(taskName0);
```

Comment:
Slightly cleaner to verify that TaskInstance's commit is called/not called instead of interacting with the OffsetManager directly (since the contract AsyncRunLoop is with TaskInstance, not OffsetManager.)